runtime/src/main/java/io/quarkiverse/githubapp/runtime/github/GitHubService.java

```diff
@@ -167,6 +170,11 @@ public CachedInstallationToken load(Long installationId) throws Exception {
     }
 
     // TODO even if we have a cache for the other one, we should probably also keep this one around for a few minutes
+    @Override
+    public GitHub getApplicationClient() {
+        return createApplicationGitHub();
+    }
```

Comment:
Yeah, if you want to use it directly now, we might need a cache. But given the duration of the TTL, I think a cache wouldn't work very well.
So I would leave it at that for now.

Reply:
FWIW, in my use case I will probably not retrieve the application client more than a few times a day, so a cache wouldn't change anything.